go/pkg/pwdb/migrations.go

```diff
@@ -146,6 +146,24 @@ func createFirstEntities(tx *gorm.DB, sfn *snowflake.Node, opts Opts) error {
 	bundle := `version: "3.7"
 networks: {}
 volumes: {}
+services:
+		gotty:
+				ports:
+					- "8080"
+				build: /home/zaraki/development/challenge-debug/gotty
```

Comment:
hardcoded path = 💀

Reply:
arg 💀💀💀💀💀💀💀💀💀💀💀💀

Reply:
and btw, you need `image: pathwar/debug@sha256:xxx` instead of `build:`, look at the other example below